github-collaboration/github-collaboration.md

```diff
@@ -38,7 +38,7 @@ In the industry, apps are created by teams of developers rather than individual
 ### Things to Remember
 - If you mess up, you can usually undo what you just did. https://github.com/blog/2019-how-to-undo-almost-anything-with-git
 - When working on a team project, NEVER push to the GitHub master branch directly. It's best to never commit directly to master as well.
-- The best GitHub practices allow us to minimizes chances for merge conflicts (this happens when multiple people worked on the same file)
+- The best GitHub practices allow us to minimizes chances for merge conflicts (this happens when multiple people worked on the same file).
```

Comment:
minimize

Reply:
I don't have to fix these things.

Reply:
I have created test cases on repl.it so for just the sake of PR,  I have made some changes here.

Reply:
Always look at the issue @Kundan28 😬

Reply:
I know that @alodahl 
You already said before that this will only be reviewed by Admins.
But the typo that @imskr fixed should be correct too. It doesn't mean that we should ignore the typos.
I hope you understand what I meant to say.

Reply:
Oh, I see.   All Shubham did was add a "."

Reply:
Done.